File: examples/leetcode-out/dart/27/remove-element.dart
int removeElement(nums, int val) {
	dynamic k = 0;
	dynamic i = 0;
	while ((i < nums.length)) {
		if ((nums[i] != val)) {
			nums[k] = nums[i];
			k = ((k + 1)).toInt();
		}
		i = ((i + 1)).toInt();
	}
	return k;
}

void main() {
}

File: examples/leetcode-out/cpp/27/remove-element.cpp
#include <bits/stdc++.h>
using namespace std;

int removeElement(vector<int> nums, int val){
	auto k = 0;
	auto i = 0;
	while (i < nums.size()) {
		if (nums[i] != val) {
			nums[k] = nums[i];
			k = k + 1;
		}
		i = i + 1;
	}
	return k;
}

int main() {
	return 0;
}
